src/client/feed/Page.js

```diff
@@ -57,6 +57,7 @@ class Page extends React.Component {
       <div>
         <Helmet>
           <title>Busy</title>
+          <meta name="robots" content="noindex" />
```

Comment:
Can you use `noindex,follow`, so Google go through sub links?

Reply:
There is no such rule as `follow`. If you want to disable following links you use `nofollow` - following links is default.

Reply:
I always used `index,follow`, Medium is using it too `view-source:https://medium.com/byteball/byteball-grants-program-906a71b93d3c`. Unless Google said it doesn't matter at all, it's more safe to have it dont you think?